addons/website_sale/static/src/js/website_sale.js

```diff
@@ -209,6 +236,17 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Set the url hash from the checked inputs and
```

Comment:
- `Sets`
- Continue typing till 80 characters

In fact : "Sets the URL hash from the selected product options." seems clear enough to me.